pkg/binding/bindingpolicy.go

```diff
@@ -124,6 +125,59 @@ func (c *Controller) getBindingPolicyByName(name string) (runtime.Object, error)
 	return got, nil
 }
 
+func (c *Controller) evaluateBindingPoliciesForUpdate(ctx context.Context, l1 labels.Set, l2 labels.Set) {
```

Comment:
If l1 and l2 can be renamed to something that gives more clarity on their sources and correlation to evaluating binding policies for updates, it would be clearer.

Reply:
Technically l1 and l2 are interchangeable from the point of view of the logic which is why it might be preferable to avoid mentioning their source.  But I renamed them to oldLabels and newLabels.